main.js

```diff
@@ -390,8 +390,26 @@ class Bmw extends utils.Adapter {
             },
             native: {},
           });
+          await this.extractKeys(this, vin + element.path + dateFormatted, data);
```

Comment:
This will lead to infinite datapoints if you add every day a new channel

Reply:
I have to admit that in nmpjs and iobroker I am a newbie.
This is the original line of your source code prepended with an 'await'.
Is this the consequence of the await?

-

Reply:
No its not a nodejs or iobroker issue.
It was a misreading by me. I already creating for every session a datapoint which fills up the iobroker. But so far no complaints